### ISSUE TITLE
GO terms with EC:1.5.3.17 xref ("non-specific polyamine oxidase")

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-ontology#25837]
Relates to #24701.

We have several specific GO terms with EC:1.5.3.17 ("non-specific polyamine oxidase") as an xref and synonym attribution (listed below). I think:
- these current EC:1.5.3.17 xrefs should be removed (or at least marked as 'broad' xrefs)
- the EC:1.5.3.17-related synonyms should be removed 
- consider adding EC:1.5.3.17 as an xref to the parent grouping term in GO (GO:0046592 polyamine oxidase activity) instead 

-----------------
id: GO:0052899
name: N(1),N(12)-diacetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity
def: "Catalysis of the reaction: H2O + N(1),N(12)-diacetylspermine + O2 = 3-acetamidopropanal + H2O2 + N(1)-acetylspermidine." [RHEA:25868]
synonym: "N(1)-acetylpolyamine oxidase activity" BROAD [EC:1.5.3.13]
synonym: "polyamine oxidase activity" BROAD [EC:1.5.3.13, **EC:1.5.3.17**]
xref: EC:1.5.3.13
xref: MetaCyc:RXN-10460
xref: RHEA:25868
is_a: GO:0046592 ! polyamine oxidase activity

id: GO:0052904
name: N1-acetylspermidine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity
def: "Catalysis of the reaction: H2O + N(1)-acetylspermidine + O2 = 3-acetamidopropanal + H2O2 + putrescine." [RHEA:25812]
synonym: "N(1)-acetylpolyamine oxidase activity" BROAD [EC:1.5.3.13]
**synonym: "non-specific polyamine oxidase activity" BROAD [EC:1.5.3.17]**
synonym: "polyamine oxidase activity" BROAD [EC:1.5.3.13, **EC:1.5.3.17**]
xref: EC:1.5.3.13
**xref: EC:1.5.3.17**
xref: KEGG_REACTION:R09074
xref: MetaCyc:RXN-12091
xref: MetaCyc:RXN-9942
xref: RHEA:25812
is_a: GO:0046592 ! polyamine oxidase activity

id: GO:0052903
name: N1-acetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity
def: "Catalysis of the reaction: H2O + N(1)-acetylspermine + O2 = 3-acetamidopropanal + H2O2 + spermidine." [RHEA:25800]
synonym: "N(1)-acetylpolyamine oxidase activity" BROAD [EC:1.5.3.13]
**synonym: "non-specific polyamine oxidase activity" BROAD [EC:1.5.3.17]**
synonym: "polyamine oxidase activity" BROAD [EC:1.5.3.13, **EC:1.5.3.17**]
xref: EC:1.5.3.13
**xref: EC:1.5.3.17**
xref: KEGG_REACTION:R03899
xref: MetaCyc:RXN-12090
xref: MetaCyc:RXN-9940
xref: RHEA:25800
is_a: GO:0046592 ! polyamine oxidase activity

id: GO:0052902
name: spermidine:oxygen oxidoreductase (3-aminopropanal-forming) activity
def: "Catalysis of the reaction: H2O + O2 + spermidine = 3-aminopropanal + H2O2 + putrescine." [RHEA:25808]
synonym: "N(1)-acetylpolyamine oxidase activity" BROAD [EC:1.5.3.13]
**synonym: "non-specific polyamine oxidase activity" BROAD [EC:1.5.3.17]**
synonym: "polyamine oxidase activity" BROAD [EC:1.5.3.13, **EC:1.5.3.17**]
xref: EC:1.5.3.13
**xref: EC:1.5.3.17**
xref: KEGG_REACTION:R09077
xref: MetaCyc:RXN-10461
xref: MetaCyc:RXN-12089
xref: RHEA:25808
is_a: GO:0046592 ! polyamine oxidase activity

id: GO:0052901
name: spermine:oxygen oxidoreductase (spermidine-forming) activity
def: "Catalysis of the reaction: H2O + O2 + spermine = 3-aminopropanal + H2O2 + spermidine." [RHEA:25804]
synonym: "N(1)-acetylpolyamine oxidase activity" BROAD [EC:1.5.3.13]
**synonym: "non-specific polyamine oxidase activity" BROAD [EC:1.5.3.17]**
synonym: "polyamine oxidase activity" BROAD [EC:1.5.3.13, **EC:1.5.3.17**]
synonym: "spermine oxidase activity" BROAD [EC:1.5.3.16]
xref: EC:1.5.3.13
xref: EC:1.5.3.16
**xref: EC:1.5.3.17**
xref: KEGG_REACTION:R09076
xref: MetaCyc:1.5.3.17-RXN
xref: MetaCyc:RXN-9015
xref: RHEA:25804
is_a: GO:0046592 ! polyamine oxidase activity

**Comment by sjm41** [comment]
@pgaudet OK if I proceed with this one?

**Comment by pgaudet** [comment]
Do we need the children? 

[GO:0052899 N(1),N(12)-diacetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0052899#display-lineage-tab)
[GO:0052904 N1-acetylspermidine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0052904#display-lineage-tab)
 [GO:0052903 N1-acetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0052903#display-lineage-tab)
 [GO:0052895 N1-acetylspermine:oxygen oxidoreductase (N1-acetylspermidine-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0052895#display-lineage-tab)
 [GO:0052893 N1-acetylspermine:oxygen oxidoreductase (propane-1,3-diamine-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0052893#display-lineage-tab)
[GO:0052897 N8-acetylspermidine:oxygen oxidoreductase (propane-1,3-diamine-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0052897#display-lineage-tab)
 [GO:0052894 norspermine:oxygen oxidoreductase activity](https://amigo.geneontology.org/amigo/term/GO:0052894#display-lineage-tab)
 [GO:0052896 spermidine oxidase (propane-1,3-diamine-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0052896#display-lineage-tab)
[GO:0052902 spermidine:oxygen oxidoreductase (3-aminopropanal-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0052902#display-lineage-tab)
 [GO:0052900 spermine oxidase (propane-1,3-diamine-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0052900#display-lineage-tab)
 [GO:0052901 spermine:oxygen oxidoreductase (spermidine-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0052901#display-lineage-tab)

Looking at the annotations, there are many instances of the same protein, it seems this may represent specific substrates?

**Comment by sjm41** [comment]
Ah, yes I agree they children could just go.
The only EXP annotations to the children are to 5 rice proteins from 3 papers:

39947 Oryza sativa subsp. japonica 	UniProt 
UniProtKB:Q0J290 	PAO7 	enables 	GO:0052894	norspermine:oxygen oxidoreductase activity PMID:24634478 		
UniProtKB:Q0J290 	PAO7 	enables 	GO:0052901	spermine:oxygen oxidoreductase (spermidine-forming) activity	PMID:24634478 		
UniProtKB:Q0J290 	PAO7 	enables 	GO:0052903	N1-acetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity	PMID:24634478 		
UniProtKB:Q0J290 	PAO7 	enables 	GO:0052904	N1-acetylspermidine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity	PMID:24634478
		
UniProtKB:Q0J954 	PAO5 	enables 	GO:0052894	norspermine:oxygen oxidoreductase activity	PMID:21796433 		
UniProtKB:Q0J954 	PAO5 	enables 	GO:0052901	spermine:oxygen oxidoreductase (spermidine-forming) activity	PMID:21796433 	
	
UniProtKB:Q5NAI7 	PAO1 	enables 	GO:0052894	norspermine:oxygen oxidoreductase activity	PMID:24105034 		
UniProtKB:Q5NAI7 	PAO1 	enables 	GO:0052901	spermine:oxygen oxidoreductase (spermidine-forming) activity	PMID:24105034 		
UniProtKB:Q5NAI7 	PAO1 	enables 	GO:0052903	N1-acetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity	PMID:24105034 
		
UniProtKB:Q7X809 	PAO3 	enables 	GO:0052894	norspermine:oxygen oxidoreductase activity	PMID:21796433 	
UniProtKB:Q7X809 	PAO3 	enables 	GO:0052901	spermine:oxygen oxidoreductase (spermidine-forming) activity	PMID:21796433 	
UniProtKB:Q7X809 	PAO3 	enables 	GO:0052903	N1-acetylspermine:oxygen oxidoreductase (3-acetamidopropanal-forming) activity	PMID:21796433 	

UniProtKB:Q7XR46 	PAO4 	enables 	GO:0052894	norspermine:oxygen oxidoreductase activity	PMID:21796433 	
UniProtKB:Q7XR46 	PAO4 	enables 	GO:0052901	spermine:oxygen oxidoreductase (spermidine-forming) activity	PMID:21796433

**Comment by sjm41** [comment]
Looks like this issue was reviewed and largely fixed as part of  https://github.com/geneontology/go-ontology/issues/28070.

GO:0052899, GO:0052904 and GO:0052902 are now obsolete.

And here's how things look with the extant terms:

```polyamine oxidase activity (GO:0046592) [EC:1.5.3.17 {source="skos:narrowMatch"}]
	|__N8-acetylspermidine:oxygen oxidoreductase (propane-1,3-diamine-forming) activity (GO:0046592) [EC:1.5.3.15]
	|__spermine oxidase (propane-1,3-diamine-forming) activity (GO:0052900) [EC:1.5.3.14]
	|__spermine oxidase activity (GO:0052901) [EC:1.5.3.16] [EC:1.5.3.17 {source="skos:broadMatch"}]
	|__N(1)-acetylpolyamine oxidase (3-acetamidopropanal-forming) activity (GO:0052903) [EC:1.5.3.13] [EC:1.5.3.17 {source="skos:broadMatch"}]
	|__thermospermine oxidase activity (GO:1990534) [no EC]
```

But I'm not sure why EC:1.5.3.17 ("non-specific polyamine oxidase" aka "polyamine oxidase") is narrowMatch on the parent and broadMatch on a couple of the children. 
The Comment on EC:1.5.3.17 say "The specific polyamine oxidases are classified as EC:1.5.3.13, EC:1.5.3.14, EC:1.5.3.15 and EC:1.5.3.16"

So, I'll make EC:1.5.3.17 exactMatch on the parent, and remove it from the two children terms.
Then I'll close this ticket.

**Comment by pgaudet** [comment]
Hi @sjm41 

The reason why it was a narrow match is that we had said that a broad specificity x is a type_of x: for example
- sugar transporter
- - glucose transporter
- - broad specificity sugar transporter

because the meaning of the parent is that it can be any of the specific children A broad specificity sugar transporter is different from a grouping class sugar transporter. But maybe that's too subtle a difference. Also, we haven't much created 'broad specificity x activity' terms (I was a bit worried about that being confusing). 
Maybe we should re-discuss this at an ontology call to make sure we're clear.

**Comment by sjm41** [comment]
Thanks for that explanation!

I understand the difference, and I can appreciate why EC has a 'broad specificity' entry alongside substrate-specific entries, but I was not aware that the GO tries to replicate that aspect (if indeed, it does - sounds like this does need to be clarified!)  IMO, trying to make that distinction in GO would indeed be too subtle/confusing. Certainly, my own understanding was that such broad specificity EC entries equated to grouping classes in GO...

**Comment by sjm41** [comment]
From discussion on editors call on 31st Mar:
- change EC:1.5.3.17 back to narrowMatch for the reasons Pascale said.

**Comment by pgaudet** [comment]
I dont think we said that ? I think 1 narrow match is enough; what do you think ?

**Comment by sjm41** [comment]
> I dont think we said that ? I think 1 narrow match is enough; what do you think ?

No, we didn't say that, and I also I think one EC:1.5.3.17 narrow match (on the parent) is sufficient!

Actually, I meant, do we want  EC:1.5.3.17 as **broadMatch** on the children terms?

Part of my edit in https://github.com/geneontology/go-ontology/issues/25837#issuecomment-2711337864 was to remove the EC:1.5.3.17 as broadMatch on two of the four children.

But I'll just go ahead and change EC:1.5.3.17 to narrowMatch on the parent, and continue to omit this xref from the child terms.

**Comment by sjm41** [comment]
Yep, that's what I've done.
